pages/mobile/account_settings.py

```diff
@@ -41,13 +43,22 @@ def click_logout(self):
         return Home(self.testsetup)
 
     def login(self, user=None):
-        fxa = self.click_sign_in()
-        fxa.login_user(user)
+        self.credentials = isinstance(user, MockUser) and user or \
```

Comment:
Same comment as above here. I really don't think that `login` should create a user.